script/vsts/get-release-version.js

```diff
@@ -4,8 +4,10 @@ const request = require('request-promise-native');
 const repositoryRootPath = path.resolve(__dirname, '..', '..');
 const appMetadata = require(path.join(repositoryRootPath, 'package.json'));
 
-const REPO_OWNER = process.env.REPO_OWNER;
-const NIGHTLY_RELEASE_REPO = process.env.NIGHTLY_RELEASE_REPO;
+const REPO_OWNER = process.env.REPO_OWNER ? process.env.REPO_OWNER : 'atom';
+const NIGHTLY_RELEASE_REPO = process.env.NIGHTLY_RELEASE_REPO
+  ? process.env.NIGHTLY_RELEASE_REPO
+  : 'atom-nightly-releases';
 
 const yargs = require('yargs');
```

Comment:
Is there a reason that MAIN_REPO is missing here?

Reply:
The script authors didn't use "the name of the main repo" anywhere in the script. I suppose they had no need for it.

And the linter gets upset if you define a variable and don't use it.

Reply:
https://github.com/atom-ide-community/atom/blob/23ad7fe8575e8419e08ede6e2ebe110e8708e725/script/vsts/get-release-version.js#L32

This is the only place in the file any of these vars is needed.

Reply:
By the way, from reading that line, it appears if our Nightly repo goes stale, and lacks recent tags, our auto drafts of releases (if we use that feature) will not advance their version number. So we would perhaps have to manually fix the suggested version number?

Something to keep in mind. Some of Atom's release CI steps are very interconnected with the Nightly repo.